initializr-service/src/main/resources/application.yml

```diff
@@ -1253,6 +1261,38 @@ initializr:
           versionRange: "[1.3.0.RELEASE,2.0.0.M1)"
           groupId: org.springframework.boot
           artifactId: spring-boot-actuator-docs
+    - name: Spring Cloud GCP
+      bom: spring-cloud-gcp
+      versionRange: 2.0.0.RC2
```

Comment:
`2.0.0.RELEASE` is out and it is a new artifact so I wouldn't bother adding RC2 here, `2.0.0.RELEASE` is better.